test/integration/targets/ios_ntp/tests/cli/ntp_remove.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Configure ntp
+  ios_ntp:
+    server:
+    source_int: Loopback0
+    acl: NTP_ACL
+    logging: true
+    key_id: 10
+    auth_key: 15435A030726242723273C21181319000A
+    auth: true
+    state: absent
```

Comment:
@Qalthos  Can we say that, since `absent` and `present` states and idempotency are tested in `ntp_configuration.yml` this play is redundancy and not needed?

Reply:
Yes, that is what I was implying from my comments.

Reply:
Great! Thanks